spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/store/JwtAccessTokenConverter.java

```diff
@@ -75,6 +65,8 @@
 
 	private AccessTokenConverter tokenConverter = new DefaultAccessTokenConverter();
 
+	private JwtClaimsSetVerifier jwtClaimsSetVerifier;
```

Comment:
Rather than doing null checks, create a `NoOpJwtClaimsSetVerifer` (private scope) and does nothing for verification